truss/truss_handle.py

```diff
@@ -189,6 +184,33 @@ def docker_run(
 
         return container
 
+    def predict(
```

Comment:
it would be nice to add a test for this with switch the `use_docker` flag

Reply:
+1 on `use_docker` test just to be sure